Altis_Life.Altis/core/functions/fn_fetchVehInfo.sqf

```diff
@@ -21,37 +21,28 @@
     11: Max Horse power
     12: Fuel Capacity
 */
-private ["_class","_scope","_picture","_displayName","_vehicleClass","_side","_faction","_superClass","_speed","_armor","_seats","_hp","_fuel"];
-_class = [_this,0,"",[""]] call BIS_fnc_param;
+params [
+    ["_class","",[""]]
+];
 if (_class isEqualTo "") exitWith {[]}; //Bad class passed.
 if (!isClass (configFile >> "CfgVehicles" >> _class)) exitWith {[]}; //Class doesn't exist in CfgVehicles
```

Comment:
My only comment is regarding this and line 31, just for the sakes of not repeating code, you could define _config beforehand, then test it with isClass (this would make it slightly slower if there is no class, but in most cases it'll be a tiny gain.).